sdk/openai/azure-ai-openai/src/main/resources/META-INF/azure-ai-openai_apiview_properties.json

```diff
@@ -0,0 +1,167 @@
+    "com.azure.ai.openai.models.ImageGenerationStyle": "Azure.OpenAI.ImageGenerationStyle",
+    "com.azure.ai.openai.models.ImageGenerations": "Azure.OpenAI.ImageGenerations",
+    "com.azure.ai.openai.models.ImageSize": "Azure.OpenAI.ImageSize",
+    "com.azure.ai.openai.models.MaxTokensFinishDetails": "Azure.OpenAI.MaxTokensFinishDetails",
+    "com.azure.ai.openai.models.OnYourDataAccessTokenAuthenticationOptions": "Azure.OpenAI.OnYourDataAccessTokenAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataApiKeyAuthenticationOptions": "Azure.OpenAI.OnYourDataApiKeyAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataAuthenticationOptions": "Azure.OpenAI.OnYourDataAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataAuthenticationType": "Azure.OpenAI.OnYourDataAuthenticationType",
+    "com.azure.ai.openai.models.OnYourDataConnectionStringAuthenticationOptions": "Azure.OpenAI.OnYourDataConnectionStringAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataDeploymentNameVectorizationSource": "Azure.OpenAI.OnYourDataDeploymentNameVectorizationSource",
+    "com.azure.ai.openai.models.OnYourDataEncodedApiKeyAuthenticationOptions": "Azure.OpenAI.OnYourDataEncodedApiKeyAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataEndpointVectorizationSource": "Azure.OpenAI.OnYourDataEndpointVectorizationSource",
+    "com.azure.ai.openai.models.OnYourDataKeyAndKeyIdAuthenticationOptions": "Azure.OpenAI.OnYourDataKeyAndKeyIdAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataModelIdVectorizationSource": "Azure.OpenAI.OnYourDataModelIdVectorizationSource",
+    "com.azure.ai.openai.models.OnYourDataSystemAssignedManagedIdentityAuthenticationOptions": "Azure.OpenAI.OnYourDataSystemAssignedManagedIdentityAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataUserAssignedManagedIdentityAuthenticationOptions": "Azure.OpenAI.OnYourDataUserAssignedManagedIdentityAuthenticationOptions",
+    "com.azure.ai.openai.models.OnYourDataVectorizationSource": "Azure.OpenAI.OnYourDataVectorizationSource",
+    "com.azure.ai.openai.models.OnYourDataVectorizationSourceType": "Azure.OpenAI.OnYourDataVectorizationSourceType",
+    "com.azure.ai.openai.models.PineconeChatExtensionConfiguration": "Azure.OpenAI.PineconeChatExtensionConfiguration",
+    "com.azure.ai.openai.models.PineconeChatExtensionParameters": "Azure.OpenAI.PineconeChatExtensionParameters",
+    "com.azure.ai.openai.models.PineconeFieldMappingOptions": "Azure.OpenAI.PineconeFieldMappingOptions",
+    "com.azure.ai.openai.models.SpeechGenerationOptions": "Azure.OpenAI.SpeechGenerationOptions",
+    "com.azure.ai.openai.models.SpeechGenerationResponseFormat": "Azure.OpenAI.SpeechGenerationResponseFormat",
+    "com.azure.ai.openai.models.SpeechVoice": "Azure.OpenAI.SpeechVoice",
+    "com.azure.ai.openai.models.StopFinishDetails": "Azure.OpenAI.StopFinishDetails"
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
Thanks Shawn. Will see if json output by codegen could ensure it has the ending newline.